### DISCUSSION TITLE
Collect and report web vitals

### DISCUSSION THREAD

**Discussion by lightningspirit** [umami-software/umami#739] [Ideas]
Hello all,

First of all, thank you for this amazing tool that works perfectly!
It came to my mind that Umami could be collect and report web vitals that SPAs send.

There are already a plenty of tools that do the initial job of collecting web vitals from SPAs (https://github.com/GoogleChrome/web-vitals and even NextJS https://nextjs.org/docs/advanced-features/measuring-performance), we need to implement an endpoint that collect an object and a graph that shows aggregated numbers.

Would you guys be interested in this feature?
 
Just for reference, the object could be something like this: 
```js
{
  id: "1626889133596-8117358425873",
  category: "web-vital",
  name: "LCP",
  value: 1979.799
}
```
Here's what the graph would be like:
https://user-images.githubusercontent.com/326742/101584324-3f9a0900-3992-11eb-8f2d-182f302fb67b.png

**Comment by kwiat1990** [comment]
Any update on this one?
